incubator/hnc/test/e2e/hns_test.go

```diff
@@ -0,0 +1,14 @@
+package e2e
+
+import (
+	. "github.com/onsi/ginkgo"
+	. "sigs.k8s.io/multi-tenancy/incubator/hnc/pkg/testutils"
+)
+
+var _ = Describe("HNS set-config", func() {
+	It("Should use '--force' flag to change from 'Ignore' to 'Propagate'", func() {
```

Comment:
You should test that none of the other transitions require `--force`, eg Ignore->Remove, Remove->Propagate, Propagate->anything. Or maybe that's overkill and at least verify that _one_ of them doesn't require `--force`.

Reply:
done